tvision/include/compatlayer.h

```diff
@@ -2108,7 +2109,9 @@ CLY_CFunc int  CLY_getcurdir(int drive, char *buffer);
  /* Platforms where sys/types.h doesn't define ssize_t: */
  #if defined(Uses_CLY_ssize_t) && !defined(CLY_ssize_t)
   #define CLY_ssize_t 1
-  typedef long ssize_t;
+  #ifndef _SSIZE_T_DEFINED
+   typedef long ssize_t;
+  #endif	
```

Comment:
I think is more clear to just avoid defining `Uses_CLY_ssize_t` at line 560. Change:

```c++
  #undef  Uses_CLY_ssize_t
  #define Uses_CLY_ssize_t 1
```
By

```c++
  #ifndef _SSIZE_T_DEFINED
    #undef  Uses_CLY_ssize_t
    #define Uses_CLY_ssize_t 1
  #endif
```